api/all/src/main/java/io/opentelemetry/api/metrics/ObservableDoubleCounter.java

```diff
@@ -14,4 +14,4 @@
  * <p>This interface currently has no methods but may be extended in the future with functionality
  * such as canceling the observable.
  */
-public interface ObservableDoubleCounter {}
+public interface ObservableDoubleCounter extends ObservableInstrument {}
```

Comment:
Why not using something more java like "AutoClosable"?

Reply:
Yup let's use AutoCloseable

Reply:
Agree with using `AutoCloseable`, but just want to mention that its a bit difficult to imagine a scenario where you'd be able to take advantage of try-with-resources with one of these async instruments. 

Still good to use `AutoCloseable` though because it's the idiomatic way to indicate that the class should be cleaned up after use.